resources/views/hardware/bulk.blade.php

```diff
@@ -20,6 +20,18 @@
     <p>{{ trans('admin/hardware/form.bulk_update_help') }}</p>
 
 
+    <div class="callout callout-warning">
+      @if($settings->full_multiple_companies_support == 1)
```

Comment:
Hm...I got `ErrorException Undefined variable $settings` from this line.

![error being shown](https://github.com/snipe/snipe-it/assets/1141514/776d7cc0-64cb-44eb-8578-928dfaa2824b)

Gonna hold off on a full review for now (unless it is something I missed).